inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/Path.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.kb.querybuilder;
+
+import org.eclipse.rdf4j.sparqlbuilder.core.QueryElement;
+import org.eclipse.rdf4j.sparqlbuilder.core.SparqlBuilder;
+import org.eclipse.rdf4j.sparqlbuilder.rdf.RdfPredicate;
+
+/**
+ * Construct property paths for use with the {@link SparqlBuilder}
+ */
+public class Path
+{
+    public static RdfPredicate of(QueryElement... aElements)
+    {
+        return () -> {
+            StringBuilder sb = new StringBuilder();
+            for (QueryElement element : aElements) {
+                if (sb.length() > 0) {
+                    sb.append("/");
+                }
+                sb.append(element.getQueryString());
+            }
+            return sb.toString();
+        };
+    }
+
+    public static QueryElement zeroOrMore(QueryElement aElement)
```

Comment:
I like javadoc for public methods

Reply:
I'm sure it will get when it's used more. It also doesn't have unit tests so far.